lib/tlscontext.c

```diff
@@ -386,6 +413,10 @@ tls_context_setup_session(TLSContext *self)
         }
       else
         msg_debug("empty ssl options");
+
+      if (!tls_context_setup_ecdh(self->ssl_ctx, SN_X9_62_prime256v1))
+          goto error;
```

Comment:
AFAIK it is available. It will be easier to give a configuration option, if it is a parameter.